docs/source/models/cause.rst

```diff
@@ -337,14 +336,38 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
```

Comment:
In the mortality impacts section, we want to put the equations we talked about.  

mort_rate_i = ACMR - Sum_causes(CSMR) + Sum_causes(EMR(X_i))